client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-list-panel.scss

```diff
@@ -16,6 +18,12 @@
 	transform: translateX(0);
 	scrollbar-gutter: stable;
 	font-family: "SF Pro Text", $sans;
+
+	@media ( max-width: $break-wide ) {
+		margin-inline-start: 300px;
+		width: 340px;
```

Comment:
How about making it smaller as the sidebar (300px)? Then the large preview might look better with a 960px window size.

Or making [the side menu open over the site preview](https://github.com/Automattic/wp-calypso/pull/77962#issuecomment-1591692260) might be a good choice as well 🙂

Reply:
I think we should not make the patterns panel smaller because we got feedback about the patterns being too small. About making the side menu open over the site preview, I don't think it is a better approach because we just removed that because we got feedback that can frustrate users. Let's find a better UX.

Reply:
Okay!